### DISCUSSION TITLE
I use the Loading component in Firefox and it works fine, but in Google and Edge, it appears as squares.

### DISCUSSION THREAD

**Discussion by szhdub** [saadeghi/daisyui#2035] [Q&A] [answered]
[https://github.com/szhdub/gleam-web-template](url)
Help me,please!,thanks!
on the firefox
![2023-06-21_16-14](https://github.com/saadeghi/daisyui/assets/23500045/c84384be-68c2-43cf-be9b-0b1401aa9439)
on the other
![2023-06-21_16-13](https://github.com/saadeghi/daisyui/assets/23500045/246b7828-f880-4af6-80b4-c8680548cf2d)

**Comment by saadeghi** [comment, marked as answer]
I can't see your repo. Is it private?  

Make sure you have `autoprefixer` plugin for PostCSS because without that plugin your CSS will not have required prefixes for different browsers. Not using `autoprefixer` may cause issue like this and many other issues.
